### PR TITLE
[IMP] website_sale_qty: Display quantity and price of lowest tiered option

### PR DESCRIPTION
**Pull request by kellylougheed**
- Enhancement of #153 
- Displays quantity and price of lowest tiered option in the search results.
- Available as an option in the "Customize" dropdown menu.

### CODE REVIEW

**Comment by lasley** [comment]
Restarting build - fail looks to be due to now merged OCA/server-tools#713

**Comment by lasley** [comment]
Oh whoops forgot I don't have write on this repo, so Travis won't take my command! @kellylougheed - please just amend your last commit and force push, which will trigger the build.

**Comment by kellylougheed** [comment]
Done @lasley

**Comment by lasley** [comment]
Someone please set this WIP - yet another one that will need to be come back to once we figure out the v10 tours 😖

File: website_sale_qty/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+Images
+------
+
+* Odoo Community Association: 
+  `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Oleg Bulkin <obulkin@laslabs.com>
```
Comment (lasley) [inline website_sale_qty/README.rst:59]: You're a contributor

**Comment by lasley** [comment]
@moylop260 - Runbot is building using Odoo v9 (incorrect), but Travis is building using v10 (correct). Any idea what could cause the disparity here? I've never seen this before.

**Comment by moylop260** [comment]
This PR is for branch 10.0 but the .travis.yml has VERSION=9.0
https://github.com/LasLabs/e-commerce/blob/15df57a388746ec7ebfd94db6d11e4f0fb7a5134/.travis.yml#L20

(Maybe we don't have covered this case for cache odoo use from runbot vs travis odoo use without cache)

**Comment by lasley** [comment]
Wow you're right @moylop260, thanks.

I find it incredibly curious that the Travis file is not included in the diff of this PR though. It's [10.0](https://github.com/OCA/e-commerce/blob/10.0/.travis.yml#L29) on the target, so one would assume that we'd see a change here.

@kellylougheed - please update the Travis file version.

**Comment by lasley** [comment]
@kellylougheed - this PR was based on 9.0. You will need to reset to 10, do format-patch, then cherry pick your migration commit(s)

**Comment by kellylougheed** [comment]
@lasley reset to OCA/10.0, did format-patch, and cherry-picked the commits. Runbot is now working!

**Comment by lasley** [comment]
@kellylougheed lint is failing

**Comment by kellylougheed** [comment]
Thanks @lasley, lint is fixed now

**Comment by lasley** [comment]
@kellylougheed - tests are not running.

File: website_sale_qty/tests/test_js.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# # -*- coding: utf-8 -*-
+# # Copyright 2016 LasLabs Inc.
+# # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+#
+# from odoo.tests import HttpCase
```
Comment (lasley) [inline website_sale_qty/tests/test_js.py]: See [this diff](https://github.com/OCA/website/pull/329/files#diff-14507ac00444cf9322aaf5cf5981c5e9R8) for upgrading to v10

File: website_sale_qty/static/tests/js/website_sale_qty.tour.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+*    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+**/
+
+odoo.define('website_sale.tour', function(require) {
+    'use strict';
+
+    var tour = require('web.Tour');
+    var _t = require('web.core')._t;
+
+    tour.register({
```
Comment (lasley) [inline website_sale_qty/static/tests/js/website_sale_qty.tour.js]: See [this diff](https://github.com/OCA/website/pull/329/files#diff-303eb7ed9721d5e1a330c9897074c32dR5) for upgrading to v10

**Review by lasley — CHANGES_REQUESTED** [review]

**Comment by lasley** [comment]
FYI @kellylougheed - You're being affected by old version of phantomjs ([MutationObserver error](https://travis-ci.org/OCA/e-commerce/jobs/213050636#L847)). You need to make the change to the travis file that I made in my website PR

**Comment by kellylougheed** [comment]
OK, thanks @lasley!

File: .travis.yml [hunk 4]
```diff
@@ -43,6 +43,8 @@ install:
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - npm install phantomjs-prebuilt@2.1.14
+  - export PATH=${TRAVIS_BUILD_DIR}/node_modules/phantomjs-prebuilt/lib/phantom/bin:${PATH}
```
Comment (lasley) [inline .travis.yml]: This can now be reverted https://github.com/OCA/maintainer-quality-tools/pull/437#issuecomment-288486224

**Comment by lasley** [comment]
@kellylougheed - I'm confused between this and #153. How is this a forward port? #153 looks like the forward port.

**Comment by kellylougheed** [comment]
@lasley It looks like #153 was a forward port of #131 (the original website_sale_qty PR), and now this PR is a forward port of #153. Is "forward port" the right term to use here? Or should I change something?

**Comment by lasley** [comment]
Forward port means you are migrating vX to vY - where X < Y. You are just adding features here right?

**Comment by kellylougheed** [comment]
@lasley Yes -- what should I call that?

**Comment by lasley** [comment]
Really we should just add the features into the existing PR, or close that one and do everything here.

While technically we would be expanding scope of the PR, I noted it in [the comments](https://github.com/OCA/e-commerce/pull/153#issuecomment-273541589), and we followed proper procedure by utilizing tickets in order to breach original scope. This is in contrast to just adding/changing things all willy-nilly.

**Comment by lasley** [comment]
But you simply call it an "improvement" or "enhancement". This is also indicated by the [IMP] prefix you used

**Comment by lasley** [comment]
@obulkin please take this over

**Comment by obulkin** [comment]
@lasley - My recommendation at this point is to close this PR. It's diverged too far from #153, and there doesn't seem to be that much in here that can be salvaged and made useful. I can return to this topic in a future enhancement to ``website_sale_qty``.

**Comment by lasley** [comment]
Understood, thanks for the eval Oleg. Make sure we have a ticket on our end for this feature - put it two sprints from now (21)

**Pull request closed without merging**